sphinxext/photofinish/__init__.py

```diff
@@ -258,14 +260,15 @@ def visit_image(
 # parallel. That funcationality is currently commented out. If it's enabled,
 # this function should be serial.
 def process_images(img_datas: Set[ImgData]):
-    for img_data in status_iterator(
-        img_datas,
-        "generating responsive images... ",
-        "blue",
-        len(img_datas),
-        stringify_func=lambda i: str(i.dest_path.name),
-    ):
-        process_image(img_data)
+    with multiprocessing.Pool(processes=os.cpu_count() - 1) as pool:
+        for img_data in status_iterator(
+            img_datas,
+            "generating responsive images... ",
+            "blue",
+            len(img_datas),
+            stringify_func=lambda i: str(i.dest_path.name),
+        ):
+            pool.apply_async(process_image, args=(img_data))
```

Comment:
I think this will sync up sphinx's status print with the completion of image conversions.
```suggestion
        results = []
        for img_data in img_datas:
            future = pool.apply_async(process_image, args=(img_data))
            results.append((future, str(img_data.dest_path.name)))
        for result in status_iterator(
            results,
            "generating responsive images... ",
            "blue",
            len(results),
            stringify_func=lambda i: i[1],
        ):
            result[0].wait()
```